src/ConfigAggregator.php

```diff
@@ -298,10 +300,13 @@ private function cacheConfig(array $config, $cachedConfigFile)
             var_export($config, true)
         ));
 
-        rename($tempFile, $cachedConfigFile);
+        // Windows doesn't like renaming an open file, so copy and unlink
+        copy($tempFile, $cachedConfigFile);
```

Comment:
This is not atomic. Same race condition applies

Reply:
Right.

About all I can think of is to either replace the `fopen()`, `fputs()` etc with a `file_put_contents()` or scrap the whole thing, switch to safe-writer and get that setting the permissions before actually writing the file. 

Any suggestions / preferences?